zerver/tests/test_events.py

```diff
@@ -340,7 +340,7 @@ def test_get_events_narrow(self):
         self.assertEqual(events[0]["message"]["display_recipient"], "Denmark")
 
 class EventsRegisterTest(ZulipTestCase):
-    user_profile = get_user_profile_by_email('hamlet@zulip.com')
+    user_profile = get_user('hamlet@zulip.com', get_realm('zulip'))
```

Comment:
Since this is a class variable, I'll move initialization of this into a new __init__ to call self.example_user